apps/transformer/lib/transform/add_timestamp.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Transform.AddTimestampField do
+  @moduledoc """
+  `Transform.Step` impl to add a timestamp field to a dataset. Timestamps will be stored in ISO-8601 format.
+
+  ## Configuration
+  * `name` - Required. Field name or path to store the timestamp under. Ex: `"timestamp"` or `["key1", "key2", "key3"]`
```

Comment:
Still being picky, if ["key1", "key2", "key3"] is a hierarchcicicacacal relationship can we say like. ["parent_field", "child_field"].  Maybe theres context in the main module i'm missing but i still dont understand how to use the second notation the way it is written.